Scooty/Scooty/RentalCompany.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Scooty
+{
+    public class RentalCompany: ScooterService, IRentalCompany
```

Comment:
RentalCompany nevajadzētu paplašināt ScooterService. :) ja nu scooter service jānomaina pret kaut ko citu? 
tad atkal jāmaina esošs kods. 
šitā nebūs labi.